server/bin/pbench-base.sh

```diff
@@ -107,7 +107,12 @@ function log_exit {
     if [[ -z "${2}" ]]; then
         exit 1
     else
-        exit ${2}
+	if [[ -z "${3}" ]]; then
+	    exit ${2}
+    	else
+	    printf -- "%b\n" "${1}" | tee -a "${3}"
```

Comment:
Indentation problems, tabs vs spaces perhaps?

It seems we might want to consider line 105 as well.  What `log_exit` used to do was send the string, `${PROG}: ${1}` to file descriptor 4.   Now we want that same string also sent to `${3}`, if that argument exists.

So you could consider something like:

```
local _msg="${PROG}: ${1}"
if [[ -z "${3}" ]]; then
    printf -- "%s\n" "${_msg}" >&4
else
    printf -- "%s\n" "${_msg}" | tee -a "${3}" >&4
fi
log_finish
if [[ -z "${2}" ]]; then
    exit 1
else
    exit ${2}
fi
```